chains/mainnet/Akash.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  chainID: 'akashnet-2',
+  lcd: 'https://akash-rpc.polkachu.com/',

```

Comment:
need LCD

Reply:
check out for REST endpoints [here](https://cosmos.directory/akash)